app/controllers/offsprings_controller.rb

```diff
@@ -0,0 +1,56 @@
+class OffspringsController < ApplicationController
+  SCOPE = "activerecord.errors.controllers.offspring".freeze # Necessary to acces locales file
```

Comment:
I was looking at this as it appeared in my code. 
IT seems that .freeze would be necessary here at the end.